config.json

```diff
@@ -692,6 +692,20 @@
     },
     {
       "core": false,
+      "difficulty": 6,
+      "slug": "tournament",
+      "topics": [
+        "loops",
+        "maps",
+        "sorting",
+        "parsing",
+        "text_formatting"
+      ],
+      "unlocked_by": "kindergarten-garden",
```

Comment:
I almost didn't notice this; this needs to be a core exercise, sorry! Exercises can only be unlocked by core exercises, and `kindergarten-garden` isn't a core exercise. Sorry for not spotting that sooner! `kindergarten-garden` is unlocked by `matrix` which is a core exercise, so maybe you want it to be unlocked by `matrix`? Or you can choose some other exercise if you want :)

Reply:
Ah, my fault for just jumping in without reading the contributing guide i guess :-)